index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/fontawesome.min.css" integrity="sha512-SgaqKKxJDQ/tAUAAXzvxZz33rmn7leYDYfBP+YoMRSENhf3zJyx3SBASt/OfeQwBHA1nxMis7mM3EV/oYT6Fdw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="./css/index.css">
+  <link rel="stylesheet" href="./css/index-media.css">
+  <title>Mosidrum</title>
+</head>
+<body>
+  <header>
+    <i class="fa-solid fa-bars" onclick="showMenu()"></i>
+    <div class="first-header">
+      <ul>
+        <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+        <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <div class="second-header">
+      <a href="index.html"><img src="./images/logo.png" alt="website_logo"></a>
+      <div class="header-right">
+        <nav>
+          <ul class="links">
+            <li><a class="nav-link active" href="about.html">About</a></li>
+            <li><a class="nav-link" href="#">Program</a></li>
+            <li><a class="nav-link" href="#">Join</a></li>
+            <li><a class="nav-link" href="#">Sponsor</a></li>
+            <i class="fa-solid fa-xmark" onclick="cancelMenu()"></i>
+          </ul>
+          <Span>Special Seat</Span>
+        </nav>
+      </div>
+    </div>
+  </header>
+  <div class="start">  
+    <div class="main">
+      <div class="main-head">
+        <p>"Hello London"</p>
+        <h1 class="animate">KING'S WORD MID-YEAR MUSICAL CONCERT <br>2023</h1>
```

Comment:
- Kindly, Your pages have to use the exact same colors, for text, backgrounds, borders, etc, from the original design as the previous code reviewer suggested to you. use those color swatch: `#272A31`, `#EC5242`, `#D3D3D3` 👌
![2023-04-21_06-12](https://user-images.githubusercontent.com/52010789/233532099-79114fd8-5ff0-447c-9902-4bd9dba026d1.png)